bokehjs/src/coffee/mapper/linear_mapper.coffee

```diff
@@ -15,9 +16,17 @@ class LinearMapper extends HasProperties
 
   v_map_to_target: (xs) ->
     [scale, offset] = @get('mapper_state')
-    result = new Float64Array(xs.length)
-    for x, idx in xs
-      result[idx] = scale * x + offset
+    if not _.isNumber(xs[0])
```

Comment:
To answer your original question:

http://underscorejs.org/#isNumber
isNumber_.isNumber(object)
Returns true if object is a Number (including NaN). 

So, in this case: `if not _.isNumber(xs[0])` is my work around for `_.isFloat64Array(xs[0])` (which doesn't existing in underscore).